.github/workflows/main.yaml

```
@@ -34,95 +34,19 @@ jobs:
       run: poetry install
 
     - name: Run tests
-      run: |
-        # test is not working
-        # make test
-        cookiecutter --output-dir /tmp/ --no-input .
-        cd /tmp/osl-python-package
-        mamba env create --file conda/dev.yaml
-        conda activate osl-python-package
-        poetry install
-        pre-commit install
-        make docs
-        make build
+      run: make test
+
+    - name: Run Smoke Test (mkdocs)
+      run: tests/smoke/mkdocs.sh
```

maybe you need to do something like
```suggestion
      run: bash tests/smoke/mkdocs.sh
```